jvm/build.sbt

```diff
@@ -16,7 +16,7 @@ Global / gatlingDevelopers := Seq(
 val compilerRelease = 21
 val graalvmJdkVersion = "23.0.1"
 val graalvmJsVersion = "24.1.1"
-val gatlingVersion = "3.13.1"
+val gatlingVersion = "3.13.2-SNAPSHOT"
```

Comment:
Waiting for published version with this feature

Reply:
Done